### ISSUE TITLE
How to train?

### ISSUE THREAD

**Issue by ErnstTmp** [SeanNaren/deepspeech.pytorch#338]
Hello Sean,
thanks a lot for creating this model, this is terrific!!!!!!

Can you comment on how you created the models in Release v1.2 that trains with a WER of 10.2 on Librispeech? Did you use any special traicks on the learning rate selection, ...?

Thanks and kind regards
Ernst

**Comment by ryanleary** [comment]
The models in Release v1.2 are unchanged from the previous release's models and how they were trained is described in release "Pre-trained models V2"
